engine/installation/linux/linux-postinstall.md

```diff
@@ -97,6 +97,69 @@ For information about the different storage engines, see
 The default storage engine and the list of supported storage engines depend on
 your host's Linux distribution and available kernel drivers.
 
+## Enable experimental features
+
+Some features of Docker are introduced as experimental features for one or more
+releases. Previously, you needed to download and install an
+**experimental build** to use these features. In Docker 1.13 and higher,
+experimental features are included in all stable Docker builds and packages,
+but disabled by default. To enable experimental features, start the Docker
+daemon (`dockerd`) with the `--experimental` flag. You can do this temporarily
+or permanently.
+
+> **Note**: Docker for Windows and Docker for Mac ship with experimental
+> features enabled by default.
+
+### Check whether experimental mode is enabled
+
+To check a running Docker daemon's configuration parameters, including whether
+experimental mode is enabled, use the `docker info` command.
+
+```bash
+$ docker info |grep Experimental
+
+Experimental: true
+```
+
+### Temporarily enable experimental features
+
+1.  Stop the Docker daemon using your operating system's commands, such as
+    `sudo service docker stop` or `sudo systemctl stop docker`.
+
+2.  Start the Docker daemon manually and add the `--experimental` flag:
+
+    ```bash
+    $ sudo /usr/bin/dockerd --experimental
+    ```
+
+The Docker daemon starts in the foreground. To stop it, use `Ctrl+C`.
+
+### Permanently enable experimental features
+
+> **Warning**: Experimental features are not appropriate for production
+> environments or workloads.
```

Comment:
Wonder if we should rephrase this, and explain why; experimental features should be as reliable as non-experimental, but design may change, or the feature as a whole may be decided to not leave experimental.